docs/narr/paste.rst

```diff
@@ -96,3 +91,8 @@ applications, servers, and :term:`middleware` defined within the configuration
 file.  The values in a ``[DEFAULT]`` section will be passed to your
 application's ``main`` function as ``global_config`` (see the reference to the
 ``main`` function in :ref:`init_py`).
+
+Alternative Configuration File Formats
+--------------------------------------
+
+It is possible to use different file formats with :app:`Pyramid` if you do not like :term:`PasteDeploy`. Under the hood all command-line scripts such as ``pserve`` and ``pshell`` pass the ``config_uri`` (e.g. ``development.ini``, ``production.ini``, etc) to the :term:`plaster` library which performs a lookup for an appropriate parser. For ``.ini`` files it uses PasteDeploy but you can register your own configuration formats that plaster will find instead.
```

Comment:
1. Many style guides use plain English over the Latin abbreviation—"for example" is preferred over "e.g.", "in other words" and "that is" over "i.e.", and "and so on" over "etc.". Reasons include:
    - the intention is clearer
    - avoids misuse of the abbreviations, usually due to the lack of knowledge of Latin
    - has a less erudite tone
    - easier to translate
2. The use of "e.g." with "etc." in the same phrase is redundant.
3. If "e.g." or "i.e." must be used, then they are followed normally by a comma in American english, and confined within parentheses.

Eventually I would like to replace Latin abbreviations with common American english in the narrative documentation so that it reads smoother.